lib/NativeCode/DynamicLibraryLoaderHelper/NativeRender/src/eos_helpers.cpp

```diff
@@ -250,19 +250,19 @@ namespace pew::eos
 
         // Get a pointer to the SteamAPI_Init function within the steam
         // library.
-        const auto SteamAPI_Init = pew::eos::eos_library_helpers::load_function_with_name<SteamAPI_Init_t>(steam_dll_handle, "SteamAPI_Init");
+        const auto SteamAPI_Init_Fn = pew::eos::eos_library_helpers::load_function_with_name<SteamAPI_Init_t>(steam_dll_handle, "SteamAPI_InitSafe");
```

Comment:
It was curious to me that this needed to be changed from `SteamAPI_Init`, because Steamworks documentation still indicates that `SteamAPI_Init` should be called, however, that function does not exist in the version of `steam_api64.dll` that is included with the steamworks plugin.

Reply:
We should add code that _falls back on_ the previous function name if the one does not exist.

Reply:
Such a fall back has now been implemented - marking this conversation as resolved.